_targets.R

```diff
@@ -19,6 +19,11 @@ param_groups_select <- c('temperature','conductivity')
 coords_lon <- c(-77.063, -75.333, -75.437)
 coords_lat <- c(40.547, 41.029, 39.880)
 
+# Specify arguments to WQP queries
+# see https://www.waterqualitydata.us/webservices_documentation for more information 
+wqp_args <- list(sampleMedia = "Water",
+                 siteType = "Stream")
```

Comment:
Oh wow, thanks for pointing this out! I used `siteType = "Stream"` here because it's what I've seen in other WQP code for pulling surface water data and because it's used in the examples for dataRetrieval (e.g. see examples at the bottom of `??dataRetrieval::whatWQPsamples`). Could this reflect a recent change in WQP? If so, that seems pretty important for folks that just want to re-run their previous scripts 😬  

If I remove the `siteType` argument from our pipeline, here's what gets returned for MonitoringLocationTypeName in the inventory:
```
> tar_load(p1_wqp_inventory)
> p1_wqp_inventory %>% group_by(MonitoringLocationTypeName) %>% summarize(n = n()) %>% filter(grepl("stream|river|channel",MonitoringLocationTypeName,ignore.case = TRUE))
# A tibble: 8 x 2
  MonitoringLocationTypeName     n
  <chr>                      <int>
1 Channelized Stream            64
2 River/Stream                6748
3 River/Stream Perennial       144
4 Riverine Impoundment          34
5 Stream                      4153
6 Stream: Canal                  2
7 Stream: Ditch                 14
8 Stream: Tidal stream           4
>
```

So it seems like "Stream" is still an allowable query value and a lot of sites are coded that way...at the least, maybe I should expand the values to include in `wqp_args` if I'm aiming for a surface water dataset containing samples from streams and rivers.

Reply:
OK, what you have there for `MonitoringLocationTypeName` matches what I was thinking originally (there are more that might match "streams") and why I went to look at the official list. It seems that the list is incomplete!

Reply:
Although, now that I've typed the above I am realizing that `MonitoringLocationTypeName` in the output and `siteType` for queries might not be 1:1. When I look at the dropdown options for `siteType` on https://www.waterqualitydata.us/, I think just "Stream" is good!

Reply:
Ah, OK! Thanks for taking the time to check this out and investigate further. I'm learning that it never hurts to double check allowable entries for WQP queries.